run.py

```diff
@@ -528,41 +528,68 @@ def speaker_info(speaker_uuid: str, core_version: Optional[str] = None):
             raise HTTPException(status_code=404, detail="該当する話者が見つかりません")
 
         try:
-            policy = (root_dir / f"speaker_info/{speaker_uuid}/policy.md").read_text(
-                "utf-8"
-            )
-            portrait = b64encode_str(
-                (root_dir / f"speaker_info/{speaker_uuid}/portrait.png").read_bytes()
-            )
-            style_infos = []
-            for style in speaker["styles"]:
-                id = style["id"]
-                icon = b64encode_str(
-                    (
-                        root_dir / f"speaker_info/{speaker_uuid}/icons/{id}.png"
-                    ).read_bytes()
-                )
-                voice_samples = [
-                    b64encode_str(
-                        (
-                            root_dir
-                            / "speaker_info/{}/voice_samples/{}_{}.wav".format(
-                                speaker_uuid, id, str(j + 1).zfill(3)
-                            )
-                        ).read_bytes()
-                    )
-                    for j in range(3)
-                ]
-                style_infos.append(
-                    {"id": id, "icon": icon, "voice_samples": voice_samples}
-                )
+            ret_data = SpeakerInfo.from_local(Speaker(**speaker))
         except FileNotFoundError:
             import traceback
 
             traceback.print_exc()
             raise HTTPException(status_code=500, detail="追加情報が見つかりませんでした")
 
-        ret_data = {"policy": policy, "portrait": portrait, "style_infos": style_infos}
+        return ret_data
+
+    @app.get("/download_infos", response_model=List[DownloadInfo], tags=["その他"])
+    def download_infos(core_version: Optional[str] = None):
+        """
+        ダウンロード可能なモデル情報を返します。
+        Returns
```

Comment:
他の所では説明とReturnsの間に1行空けています。

```suggestion
        ダウンロード可能なモデル情報を返します。

        Returns
```